JustSaying.Messaging/Interrogation/InterrogationResponse.cs

```diff
@@ -0,0 +1,16 @@
+using System.Collections.Generic;
+
+namespace JustSaying.Messaging.Interrogation
+{
+    public class InterrogationResponse : IInterrogationResponse
+    {
+        public InterrogationResponse()
+        {
+            Subscribers = new List<ISubscriber>();
+            Publishers = new List<IPublisher>();
+        }
+
+        public ICollection<ISubscriber> Subscribers { get; }
```

Comment:
The contents of list is currently mutable. This is required to allow the response is built up over the time of the setup. The list itself is immutable I cannot think of a reason to make the list mutable.